examples/chart/app.py

```diff
@@ -6,39 +6,45 @@
 from toga.constants import COLUMN
 
 
-def sample_histogram():
-    np.random.seed(19680801)
+class ExampleChartApp(toga.App):
 
-    # example data
-    mu = 100  # mean of distribution
-    sigma = 15  # standard deviation of distribution
-    x = mu + sigma * np.random.randn(437)
+    def draw_chart(self):
+        # example data
+        mu = 100  # mean of distribution
+        sigma = 15  # standard deviation of distribution
+        x = mu + sigma * np.random.randn(437)
 
-    num_bins = 50
+        num_bins = 50
 
-    f = Figure(figsize=(5, 4))
-    ax = f.add_subplot(1, 1, 1)
+        dpi = 100  # as of writing, 100 is also the default DPI for matplotlib.figure.Figure
+        f = Figure(figsize=(self.chart.layout.content_width / dpi, self.chart.layout.content_height / dpi), dpi=dpi)
```

Comment:
I'm not particularly a fan of how this accesses something that doesn't really seem to be documented (or at least somewhere I thought to look), but it's how it's done in the Canvas example for Toga:  https://github.com/beeware/toga/blob/e1650326e77d09a27f7e916072bb3379b334977f/examples/canvas/canvas/app.py#L282

Reply:
Yeah - it's not formally documented, but I'd interpret the `layout` attribute as fair game for read access.